doc/code/qml_templates.rst

```diff
@@ -53,6 +53,17 @@ predefined pattern. You can import this function both via ``qml.broadcast`` and
 
     pennylane.broadcast
 
+Repetition Function
+-------------------
+
+The repetition function creates a new template by repeatedly applying a quantum circuit (containing quantum
+gates and subroutines) to a set of wires. You can import this function both via
```

Comment:
To me, a quantum circuit is somehow the same as a quantum function, and it includes measurements by definition. Maybe better to change to:

```suggestion
The repetition function creates a new template by repeatedly applying a sequence of quantum
gates to a set of wires. You can import this function both via
```